rdt/transformers/pii/anonymizer.py

```diff
@@ -162,7 +162,7 @@ def _transform(self, data):
 
         return None
 
-    def _reverse_transform(self, data):
+    def _reverse_transform(self, data, sample_size=None):
```

Comment:
we can get rid of this parameter right? we're just using the data's length

Reply:
Forgot to remove it from the `anonymizer`